en/docs/reference/synapse-properties/synapse-expressions.md

```diff
@@ -0,0 +1,777 @@
+# Synapse Expressions
+
+## Overview
+
+Synapse Expressions is a powerful expression designed to streamline the handling of JSON payloads. Handling of XML payloads is also supported using a dedicated function to evaluate XPATH expressions.
+
+Since, JSONPath is limited to extracting JSON content only, it's necessary to use XPath for performing additional operations on the extracted data (e.g., addition, substring extraction). Synapse Expressions addresses this limitation by enabling direct operations on JSON payloads, eliminating the need for complex workarounds.
+
+Furthermore, Synapse Expressions provides a unified syntax to access a wide range of system elements, including properties, variables, configurations, headers, registry content, secrets, and parameters. This cohesive approach simplifies development, making it significantly more efficient and user-friendly for developers.
+
+## Syntax
+
+The Synapse Expression syntax is as follows:
+
+```
+${expression}
+```
+
+Here, expression can be a combination of literals, value access expressions, operators, filter expressions, conditional expressions and functions.
+
+Example usage:
+
+```
+<log level="custom">
+    <property name="Greeting" expression="${'Hello ' + payload.student.name}"/>
+</log>
+```
+
+!!! Note
+    For the rest of this document, we will refer to the Synapse Expression using the expression only. When using the expression, the expression should be enclosed within `${}`.
+
+## Literals
+
+Literals are constant values that are directly used in the expression. 
+
+The following types of literals are supported in Synapse Expressions.
+
+<table>
+<thead>
+<tr class="header">
+<th>Literal Type</th>
+<th>Example</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>String</td>
+<td><pre><code>"Hello World"</code></pre></td>
```

Comment:
Please remove the `<pre></pre>` tags from all the tables. They render inconsistently with other pages on the UI.

Reply:
fixed in https://github.com/wso2/docs-mi/pull/1152/commits/d4d78c337fca0ff9b774ace7b7f9a1abe21cf1b2